### PR TITLE
fix: caught invalid url image exception in resize image tasks

### LINKED ISSUE #6618: Image resizing is not working
https://github.com/fossasia/open-event-server/blob/9bb9f68305c261713ee486f3840ebcf55dc70d23/app/api/helpers/files.py#L80

urlib is throwing 403 Forbidden error for almost all images. 

Please use requests and verify image resizing works. Also, create a management command which finds all events which have no resized images and resizes them

### PR DESCRIPTION
**Pull request by codedsun**
Fixes #6618 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by codedsun** [comment]
@iamareebjamal Check this, but their is a similar exception for `URLRequired` shall this also be catched?

**Comment by iamareebjamal** [comment]
What does catched mean?

**Comment by iamareebjamal** [comment]
In management command, add a log showing how many events and speakers and being resized. And ID of each:

```
Resizing images of 18 events...
Resizing Event 857
Resizing Event 191
Resizing Event 774
Resizing Event 773
Resizing Event 197
Resizing Event 169
Resizing Event 170
Resizing Event 171
Resizing Event 782
Resizing Event 781
Resizing Event 780
Resizing Event 931
Resizing Event 896
Resizing Event 895
Resizing Event 936
Resizing Event 934
Resizing Event 1014
Resizing Event 1008

Resizing images of 6 speakers...
Resizing Speaker 4015
Resizing Speaker 3994
Resizing Speaker 3974
Resizing Speaker 4060
Resizing Speaker 4064
Resizing Speaker 4130

```

**Comment by codedsun** [comment]
>AttributeError("module 'requests.exceptions' has no attribute 'URLError'")

For this error I have except the exception as their was no URL Error module and have replaced it with invalid URL.

> What does catched mean?

Try Catch in java, try except in python

**Comment by codedsun** [comment]
https://github.com/fossasia/open-event-server/pull/6648#issuecomment-563158331

I will do it by some time, please give me some time. I did tested it. I told you the error is due to a typo error, check the PR you will see.

**Comment by codedsun** [comment]
@iamareebjamal Did it made sense?

**Comment by iamareebjamal** [comment]
There is no word 'catched'. It's caught

**Comment by codedsun** [comment]
> There is no word 'catched'. It's caught

Thankyou, learnt' that.

**Comment by codedsun** [comment]
@iamareebjamal tell me is thier something left to be done here?

**Comment by iamareebjamal** [comment]
https://github.com/fossasia/open-event-server/pull/6648#issuecomment-563158331

**Comment by codedsun** [comment]
What I did before running the task 
1. #UPDATE events set thumbnail_image_url=NULL, large_image_url=NULL,icon_image_url=NULL where id in (11,31,34,35)

2. UPDATE events set original_image_url='https://storage.eventyay.com/eventyay.com/events/758/background/Vk0zSTdyZX/1517e947-29df-4a3c-9580-373db3ba9ba9' where id = 32;

(Removed the images, broked a url for event 32)

For speakers - Removed the urls for all
UPDATE speaker set thumbnail_image_url=NULL, small_image_url=NULL,icon_image_url=NULL where id in (1,2)


```
[2019-12-09 19:59:17,755: INFO/MainProcess] Received task: resize.event.images[52eaabcd-b294-4e45-a44e-fcdd41055f78]  
[2019-12-09 19:59:17,768: INFO/ForkPoolWorker-1] Event image resizing tasks started http://127.0.0.1:5000/static/media/temp/images/108b4e75-8d5c-41e9-83ef-f31989ba3e03/NTlVOU1ncW/5dd74986-2538-40c2-a560-e4dd410c1ba6.png
[2019-12-09 19:59:17,770: INFO/MainProcess] Received task: resize.event.images[6a5c3a3a-85fe-469d-92c3-c9314749b3d0]  
[2019-12-09 19:59:17,776: INFO/MainProcess] Received task: resize.event.images[421ee25d-b1f5-43c9-b5cb-b349215f8d32]  
[2019-12-09 19:59:17,782: INFO/MainProcess] Received task: resize.event.images[000a9a08-3ee8-4560-874c-57b9b9fd6724]  
[2019-12-09 19:59:17,837: INFO/MainProcess] Received task: resize.speaker.images[aa0fcad7-7db7-40b3-b80f-73ae5e88adec]  
[2019-12-09 19:59:17,841: INFO/MainProcess] Received task: resize.speaker.images[0e9e3375-7caf-4eb2-8b85-f6bed73d4441]  
[2019-12-09 19:59:17,868: INFO/ForkPoolWorker-2] Event image resizing tasks started https://storage.eventyay.com/eventyay.com/events/758/background/Vk0zSTdyZX/1517e947-29df-4a3c-9580-373db3ba9ba9
[2019-12-09 19:59:17,887: INFO/ForkPoolWorker-4] Event image resizing tasks started http://127.0.0.1:5000/static/media/temp/images/ede852be-424a-479e-95b1-e4b217ef86ec/TUtSL0JsT0/9e5747a1-f246-4e58-a683-12e377c43657.png
[2019-12-09 19:59:17,896: INFO/ForkPoolWorker-3] Event image resizing tasks started http://127.0.0.1:5000/static/media/temp/images/421f86a9-72f9-471f-bea9-ba1b7da2e880/SkxNcDAvRH/5381d308-bf93-43de-b993-18d9ec102d01.png
[2019-12-09 19:59:18,828: INFO/ForkPoolWorker-1] Saved to db
[2019-12-09 19:59:18,832: INFO/ForkPoolWorker-1] added to session
[2019-12-09 19:59:18,889: INFO/ForkPoolWorker-4] Saved to db
[2019-12-09 19:59:18,892: INFO/ForkPoolWorker-4] added to session
[2019-12-09 19:59:18,903: INFO/ForkPoolWorker-1] Resized images saved successfully for event with id: 11
[2019-12-09 19:59:18,906: INFO/ForkPoolWorker-1] Task resize.event.images[52eaabcd-b294-4e45-a44e-fcdd41055f78] succeeded in 1.1494421490000377s: None
[2019-12-09 19:59:18,913: INFO/ForkPoolWorker-1] Speaker image resizing tasks started for speaker with id 1
[2019-12-09 19:59:18,921: INFO/ForkPoolWorker-3] Saved to db
[2019-12-09 19:59:18,923: INFO/ForkPoolWorker-3] added to session
[2019-12-09 19:59:19,030: INFO/ForkPoolWorker-4] Resized images saved successfully for event with id: 34
[2019-12-09 19:59:19,031: INFO/ForkPoolWorker-3] Resized images saved successfully for event with id: 35
[2019-12-09 19:59:19,035: INFO/ForkPoolWorker-4] Task resize.event.images[421ee25d-b1f5-43c9-b5cb-b349215f8d32] succeeded in 1.2469148659999973s: None
[2019-12-09 19:59:19,036: INFO/ForkPoolWorker-3] Task resize.event.images[000a9a08-3ee8-4560-874c-57b9b9fd6724] succeeded in 1.22912490300007s: None
[2019-12-09 19:59:19,047: INFO/ForkPoolWorker-4] Speaker image resizing tasks started for speaker with id 2
[2019-12-09 19:59:19,451: ERROR/ForkPoolWorker-2] Task resize.event.images[6a5c3a3a-85fe-469d-92c3-c9314749b3d0] raised unexpected: OSError('Corrupt/Invalid Image')
Traceback (most recent call last):
  File "/Users/codedsun/Documents/FOSS/open-event-server/app/api/helpers/files.py", line 84, in create_save_resized_image
    im = Image.open(image_file)
  File "/usr/local/lib/python3.7/site-packages/PIL/Image.py", line 2818, in open
    raise IOError("cannot identify image file %r" % (filename if filename else fp))
OSError: cannot identify image file <_io.BytesIO object at 0x106588950>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/celery/app/trace.py", line 385, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/flask_celeryext/app.py", line 101, in __call__
    res = Task.__call__(self, *args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/celery/app/trace.py", line 648, in __protected_call__
    return self.run(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/sentry_sdk/integrations/celery.py", line 158, in _inner
    reraise(*exc_info)
  File "/usr/local/lib/python3.7/site-packages/sentry_sdk/_compat.py", line 57, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/sentry_sdk/integrations/celery.py", line 153, in _inner
    return f(*args, **kwargs)
  File "/Users/codedsun/Documents/FOSS/open-event-server/app/api/helpers/tasks.py", line 132, in resize_event_images_task
    uploaded_images = create_save_image_sizes(original_image_url, 'event-image', event.id)
  File "/Users/codedsun/Documents/FOSS/open-event-server/app/api/helpers/files.py", line 188, in create_save_image_sizes
    'original_image_url': create_save_resized_image(image_file, 0, 0, 0, original_upload_path, resize=False),
  File "/Users/codedsun/Documents/FOSS/open-event-server/app/api/helpers/files.py", line 86, in create_save_resized_image
    raise IOError("Corrupt/Invalid Image")
OSError: Corrupt/Invalid Image
[2019-12-09 19:59:19,508: INFO/ForkPoolWorker-1] Saved to db
[2019-12-09 19:59:19,509: INFO/ForkPoolWorker-1] added to session
[2019-12-09 19:59:19,536: INFO/ForkPoolWorker-1] Resized images saved successfully for speaker with id: 1
[2019-12-09 19:59:19,537: INFO/ForkPoolWorker-1] Task resize.speaker.images[aa0fcad7-7db7-40b3-b80f-73ae5e88adec] succeeded in 0.62929151000003s: None
[2019-12-09 19:59:19,744: INFO/ForkPoolWorker-4] Saved to db
[2019-12-09 19:59:19,744: INFO/ForkPoolWorker-4] added to session
[2019-12-09 19:59:19,790: INFO/ForkPoolWorker-4] Resized images saved successfully for speaker with id: 2
[2019-12-09 19:59:19,792: INFO/ForkPoolWorker-4] Task resize.speaker.images[0e9e3375-7caf-4eb2-8b85-f6bed73d4441] succeeded in 0.7544786979999571s: None


```

**Comment by codedsun** [comment]
Next, I broke this
UPDATE events set thumbnail_image_url=NULL, large_image_url=NULL,icon_image_url=NULL where id in (34)

UPDATE events set original_image_url='suneet' where id = 34;

```

[2019-12-09 20:11:13,314: ERROR/ForkPoolWorker-2] Task resize.event.images[50481d85-7016-4ac4-9aef-8d98ae8d83d7] raised unexpected: MissingSchema("Invalid URL 'suneet': No schema supplied. Perhaps you meant http://suneet?")
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/celery/app/trace.py", line 385, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/flask_celeryext/app.py", line 101, in __call__
    res = Task.__call__(self, *args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/celery/app/trace.py", line 648, in __protected_call__
    return self.run(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/sentry_sdk/integrations/celery.py", line 158, in _inner
    reraise(*exc_info)
  File "/usr/local/lib/python3.7/site-packages/sentry_sdk/_compat.py", line 57, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/sentry_sdk/integrations/celery.py", line 153, in _inner
    return f(*args, **kwargs)
  File "/Users/codedsun/Documents/FOSS/open-event-server/app/api/helpers/tasks.py", line 132, in resize_event_images_task
    uploaded_images = create_save_image_sizes(original_image_url, 'event-image', event.id)
  File "/Users/codedsun/Documents/FOSS/open-event-server/app/api/helpers/files.py", line 188, in create_save_image_sizes
    'original_image_url': create_save_resized_image(image_file, 0, 0, 0, original_upload_path, resize=False),
  File "/Users/codedsun/Documents/FOSS/open-event-server/app/api/helpers/files.py", line 81, in create_save_resized_image
    data = requests.get(image_file).content
  File "/usr/local/lib/python3.7/site-packages/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/requests/sessions.py", line 519, in request
    prep = self.prepare_request(req)
  File "/usr/local/lib/python3.7/site-packages/requests/sessions.py", line 462, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "/usr/local/lib/python3.7/site-packages/requests/models.py", line 313, in prepare
    self.prepare_url(url, params)
  File "/usr/local/lib/python3.7/site-packages/requests/models.py", line 387, in prepare_url
    raise MissingSchema(error)
requests.exceptions.MissingSchema: Invalid URL 'suneet': No schema supplied. Perhaps you meant http://suneet?


[2019-12-09 20:11:14,473: ERROR/ForkPoolWorker-1] Task resize.event.images[4ce91433-8704-474c-92ef-5437a0e86a0a] raised unexpected: OSError('Corrupt/Invalid Image')
Traceback (most recent call last):
  File "/Users/codedsun/Documents/FOSS/open-event-server/app/api/helpers/files.py", line 84, in create_save_resized_image
    im = Image.open(image_file)
  File "/usr/local/lib/python3.7/site-packages/PIL/Image.py", line 2818, in open
    raise IOError("cannot identify image file %r" % (filename if filename else fp))
OSError: cannot identify image file <_io.BytesIO object at 0x106588950>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/celery/app/trace.py", line 385, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/flask_celeryext/app.py", line 101, in __call__
    res = Task.__call__(self, *args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/celery/app/trace.py", line 648, in __protected_call__
    return self.run(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/sentry_sdk/integrations/celery.py", line 158, in _inner
    reraise(*exc_info)
  File "/usr/local/lib/python3.7/site-packages/sentry_sdk/_compat.py", line 57, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/sentry_sdk/integrations/celery.py", line 153, in _inner
    return f(*args, **kwargs)
  File "/Users/codedsun/Documents/FOSS/open-event-server/app/api/helpers/tasks.py", line 132, in resize_event_images_task
    uploaded_images = create_save_image_sizes(original_image_url, 'event-image', event.id)
  File "/Users/codedsun/Documents/FOSS/open-event-server/app/api/helpers/files.py", line 188, in create_save_image_sizes
    'original_image_url': create_save_resized_image(image_file, 0, 0, 0, original_upload_path, resize=False),
  File "/Users/codedsun/Documents/FOSS/open-event-server/app/api/helpers/files.py", line 86, in create_save_resized_image
    raise IOError("Corrupt/Invalid Image")
OSError: Corrupt/Invalid Image


```

**Comment by codedsun** [comment]
Now I did

Next, I broke this
UPDATE events set thumbnail_image_url=NULL, large_image_url=NULL,icon_image_url=NULL where id in (34)

UPDATE events set original_image_url='http://suneet' where id = 34;

```
[2019-12-09 20:13:33,582: INFO/MainProcess] Received task: resize.event.images[bb65fdce-b2b1-4417-8b55-6bd3eb12795c]  
[2019-12-09 20:13:33,594: INFO/ForkPoolWorker-1] Event image resizing tasks started https://storage.eventyay.com/eventyay.com/events/758/background/Vk0zSTdyZX/1517e947-29df-4a3c-9580-373db3ba9ba9
[2019-12-09 20:13:33,598: INFO/MainProcess] Received task: resize.event.images[026b196d-d5a1-42d1-9de9-5f199f11b52f]  
[2019-12-09 20:13:33,610: INFO/ForkPoolWorker-2] Event image resizing tasks started http://suneet
[2019-12-09 20:13:33,796: ERROR/ForkPoolWorker-2] Task resize.event.images[026b196d-d5a1-42d1-9de9-5f199f11b52f] raised unexpected: ConnectionError(MaxRetryError("HTTPConnectionPool(host='suneet', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10c424950>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known'))"))
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/urllib3/connection.py", line 159, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw)
  File "/usr/local/lib/python3.7/site-packages/urllib3/util/connection.py", line 57, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "/usr/local/Cellar/python/3.7.5/Frameworks/Python.framework/Versions/3.7/lib/python3.7/socket.py", line 748, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 600, in urlopen
    chunked=chunked)
  File "/usr/local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 354, in _make_request
    conn.request(method, url, **httplib_request_kw)
  File "/usr/local/Cellar/python/3.7.5/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 1252, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "/usr/local/Cellar/python/3.7.5/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 1298, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "/usr/local/Cellar/python/3.7.5/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 1247, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "/usr/local/Cellar/python/3.7.5/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 1026, in _send_output
    self.send(msg)
  File "/usr/local/Cellar/python/3.7.5/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 966, in send
    self.connect()
  File "/usr/local/lib/python3.7/site-packages/urllib3/connection.py", line 181, in connect
    conn = self._new_conn()
  File "/usr/local/lib/python3.7/site-packages/urllib3/connection.py", line 168, in _new_conn
    self, "Failed to establish a new connection: %s" % e)
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPConnection object at 0x10c424950>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/requests/adapters.py", line 449, in send
    timeout=timeout
  File "/usr/local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 638, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "/usr/local/lib/python3.7/site-packages/urllib3/util/retry.py", line 399, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='suneet', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10c424950>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/celery/app/trace.py", line 385, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/flask_celeryext/app.py", line 101, in __call__
    res = Task.__call__(self, *args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/celery/app/trace.py", line 648, in __protected_call__
    return self.run(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/sentry_sdk/integrations/celery.py", line 158, in _inner
    reraise(*exc_info)
  File "/usr/local/lib/python3.7/site-packages/sentry_sdk/_compat.py", line 57, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/sentry_sdk/integrations/celery.py", line 153, in _inner
    return f(*args, **kwargs)
  File "/Users/codedsun/Documents/FOSS/open-event-server/app/api/helpers/tasks.py", line 132, in resize_event_images_task
    uploaded_images = create_save_image_sizes(original_image_url, 'event-image', event.id)
  File "/Users/codedsun/Documents/FOSS/open-event-server/app/api/helpers/files.py", line 188, in create_save_image_sizes
    'original_image_url': create_save_resized_image(image_file, 0, 0, 0, original_upload_path, resize=False),
  File "/Users/codedsun/Documents/FOSS/open-event-server/app/api/helpers/files.py", line 81, in create_save_resized_image
    data = requests.get(image_file).content
  File "/usr/local/lib/python3.7/site-packages/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python3.7/site-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/requests/adapters.py", line 516, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPConnectionPool(host='suneet', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10c424950>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known'))

[2019-12-09 20:13:34,530: ERROR/ForkPoolWorker-1] Task resize.event.images[bb65fdce-b2b1-4417-8b55-6bd3eb12795c] raised unexpected: OSError('Corrupt/Invalid Image')
Traceback (most recent call last):
  File "/Users/codedsun/Documents/FOSS/open-event-server/app/api/helpers/files.py", line 84, in create_save_resized_image
    im = Image.open(image_file)
  File "/usr/local/lib/python3.7/site-packages/PIL/Image.py", line 2818, in open
    raise IOError("cannot identify image file %r" % (filename if filename else fp))
OSError: cannot identify image file <_io.BytesIO object at 0x1052f2230>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/celery/app/trace.py", line 385, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/flask_celeryext/app.py", line 101, in __call__
    res = Task.__call__(self, *args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/celery/app/trace.py", line 648, in __protected_call__
    return self.run(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/sentry_sdk/integrations/celery.py", line 158, in _inner
    reraise(*exc_info)
  File "/usr/local/lib/python3.7/site-packages/sentry_sdk/_compat.py", line 57, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/sentry_sdk/integrations/celery.py", line 153, in _inner
    return f(*args, **kwargs)
  File "/Users/codedsun/Documents/FOSS/open-event-server/app/api/helpers/tasks.py", line 132, in resize_event_images_task
    uploaded_images = create_save_image_sizes(original_image_url, 'event-image', event.id)
  File "/Users/codedsun/Documents/FOSS/open-event-server/app/api/helpers/files.py", line 188, in create_save_image_sizes
    'original_image_url': create_save_resized_image(image_file, 0, 0, 0, original_upload_path, resize=False),
  File "/Users/codedsun/Documents/FOSS/open-event-server/app/api/helpers/files.py", line 86, in create_save_resized_image
    raise IOError("Corrupt/Invalid Image")
OSError: Corrupt/Invalid Image


```

**Comment by codedsun** [comment]
@iamareebjamal Provided you everything

**Comment by iamareebjamal** [comment]
> In management command, add a log showing how many events and speakers and being resized. And ID of each:

What have you done in the previous comments? I don't get it. I asked you to add log statements in the management command so that when I run the command, I can see how many and which events and speakers are being resized.

**Comment by codedsun** [comment]
@iamareebjamal For the event-resizing task, as I had earlier ran this command all the images were fixed. Now to remove that I changed the fields in db for events of `thumbnail_image_url=NULL, large_image_url=NULL,icon_image_url=NULL `. 

Check this, you can find the events id

https://github.com/fossasia/open-event-server/pull/6648#issuecomment-563265533

**Comment by codedsun** [comment]
@iamareebjamal Done once again. I have logged in error

```
ERROR:root:suneet hi
ERROR:root:Event resizing - 11
ERROR:root:Event resizing - 31
ERROR:root:Event resizing - 34
ERROR:root:Event resizing - 35
ERROR:root:Speakeer resizing - 1
ERROR:root:Speakeer resizing - 2
```

**Comment by iamareebjamal** [comment]
I can see no code change here. What use do the logs of your machine serve me?

**Comment by iamareebjamal** [comment]
I want to see logs like this - https://github.com/fossasia/open-event-server/pull/6648#issuecomment-563158331 when I run the management command. Right now it prints nothing. How hard it is to understand? I must be a terrible communicator.

**Comment by codedsun** [comment]
Omg! This was lit -_- you wanted me to add a log statement in this command.

File: manage.py [hunk 1]
```diff
@@ -45,12 +48,14 @@ def fix_event_and_speaker_images():
...
                                     Event.icon_image_url == None)).all()
     for event in events:
+        logger.info('Resizing Event %s', event.id)
         resize_event_images_task.delay(event.id, event.original_image_url)
 
     speakers = Speaker.query.filter(Speaker.photo_url.isnot(None),
                                     or_(Speaker.icon_image_url == None,
                                         Speaker.small_image_url == None, Speaker.thumbnail_image_url == None))
     for speaker in speakers:
+        logging.error('Resizing Speaker %s', speaker.id)
```
Comment (iamareebjamal) [inline manage.py]: Why is it error

**Comment by iamareebjamal** [comment]
Still not showing count. Look at it carefully - https://github.com/fossasia/open-event-server/pull/6648#issuecomment-563158331

File: manage.py [hunk 2]
```diff
@@ -44,13 +47,18 @@ def fix_event_and_speaker_images():
...
+    logger.info('Resizing images of  %s events...', events.count)
     for event in events:
+        logger.info('Resizing Event %s', event.id)
         resize_event_images_task.delay(event.id, event.original_image_url)
 
     speakers = Speaker.query.filter(Speaker.photo_url.isnot(None),
                                     or_(Speaker.icon_image_url == None,
                                         Speaker.small_image_url == None, Speaker.thumbnail_image_url == None))
+
+    logger.info('Resizing images of  %s speakers...', speakers.count)
```
Comment (iamareebjamal) [inline manage.py]: Will fail here

Comment (codedsun) [reply manage.py]: Yes, it's failing.

Comment (codedsun) [reply manage.py]: Debugging it

**Comment by codedsun** [comment]
@iamareebjamal Fixed, check now!

**Comment by iamareebjamal** [comment]
Change safe_query to normal query in another PR

**Pull request merged**